index.js

```diff
@@ -56,9 +55,15 @@ exports.kinesisHandler = function (records, context, callback) {
       })
 
       logger.debug({'message': 'Update records array length' + updateRecordsArray.length })
-      // if any MLN items, send them on to the MLN API
-      if (updateRecordsArray.length != 0) {
-        postRecords(updateRecordsArray, accessToken)
+      var maxRetries = 5
```

Comment:
I am merging this code to QA for testing. If anything happen I will revert it back with production file